doc/app/NRFCLOUD_AT_commands.rst

```diff
@@ -253,7 +253,7 @@ This is emitted when the location request failed, either when sending it or rece
 No notification containing location data will be emitted.
 
 * The ``<error>`` value indicates the error that happened.
-  It is either a negative *errno* code or one of the :c:enum:`nrf_cloud_error` values.
+  It is one of the :c:enum:`nrf_cloud_error` values.
```

Comment:
We won't block this PR with this comment but maybe we should look into how to add a link to these values? @divipillai 
One option is to refer to although that's not 100% accurate as the enum values in sdk-nrf may not be in synch at all times: https://api.nrfcloud.com/v1#section/Error-Codes

Another option is nrf_cloud_error in https://docs.nordicsemi.com/bundle/nrf-apis-latest/page/nrf_cloud_8h.html
Here we don't see the values: https://docs.nordicsemi.com/bundle/nrf-apis-latest/page/group_nrf_cloud_ga2c8431e00045e980f3fc282e1567928e.html#ga2c8431e00045e980f3fc282e1567928e